data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/test_overwrite_specials_filter.cfg

```diff
@@ -0,0 +1,112 @@
+#####
+# API(s) being tested: [damage]overwrite_specials=both_sides,[overwrite][filter_specials]
+##
+# Actions:
+# Three [damage] abilities are added, one with value=5, other with add=11 and the last with multiply=2.
+# The one with multiply=2 have overwrite_specials=both_sides and [overwrite][filter_specials] for overwrite [damage] with add=11 but not value=5.
+# Have alice attack bob.
+##
+# Expected end state:
+# Alice has 100 - (5*2)=90 hitpoints because add=11 overwrited but not value=5 who replace 9 pt of damage of bob.
+#####
+{GENERIC_UNIT_TEST "test_overwrite_specials_filter" (
+    [event]
+        name=start
+        [modify_unit]
+            [filter]
+            [/filter]
+            max_hitpoints=100
+            hitpoints=100
+            attacks_left=1
+        [/modify_unit]
+        [object]
+            silent=yes
+            [effect]
+                apply_to=new_ability
+                [abilities]
+                    [damage]
+                        id=test_dmg
+                        value=5
+                    [/damage]
+                    [attacks]
+                        id=test_att
+                        value=1
+                    [/attacks]
+                    [damage]
+                        id=test_overwrite
+                        add=11
+                    [/damage]
+                    [damage]
+                        id=test_overwrite_specials
+                        multiply=2
+                        overwrite_specials=both_sides
+                        [overwrite]
+                            [filter_specials]
+                                [not]
+                                    type_value=value
```

Comment:
What is `type_value`? It doesn't look like it's new for this PR, but I can't find it documented anywhere either.

Reply:
> What is `type_value`? It doesn't look like it's new for this PR, but I can't find it documented anywhere either.

 `type_value` is an attribute filter used for determine if an ability or specialuse attribite like `value`, `add`, multiply` or `divide` without determine without having to specify a value range, here it is for verify what special using value attribute, but now value range accept negative value,  `type_value` will be prehaps be obsolete.

Reply:
changed in https://github.com/wesnoth/wesnoth/pull/7746/commits/bd6824c6ffdf21e92740606ee270d13d544d525f.

Reply:
commit removed

Reply:
Alright, so what this means is: overwrite any weapon special or "weapon special as an ability" that doesn't have a `value` attribute?